lib/ansible/modules/storage/hpe3par/cpg_3par.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/python
+# Copyright: (c) 2018, Hewlett Packard Enterprise Development LP
+# GNU General Public License v3.0+
```

Comment:
Not sure if the comment must be on the same line than the license name, @gundalow any advice ?